source/_integrations/monzo.markdown

```diff
@@ -0,0 +1,94 @@
+---
+title: Monzo
+description: Instructions on how to integrate Monzo integration into Home Assistant.
+ha_category:
+  - Sensor
+ha_release: "2023.10.1"
+ha_iot_class: Cloud Polling
+ha_codeowners:
+  - '@JakeMartin-ICL'
+ha_config_flow: true
+ha_domain: monzo
+ha_homekit: true
+ha_platforms:
+  - sensor
+related:
+  - docs: /docs/configuration/remote/
+    title: Remote access to Home Assistant
+---
+
+The **Monzo** {% term integration %} allows you to connect your Monzo bank accounts to Home Assistant
+
+- [Prerequisites and Approval](#prerequisites-and-approval)
+  - [Adding a second account](#adding-a-second-account)
+- [Sensor](#sensor)
+- [Webhooks and Triggers](#webhooks-and-triggers)
+- [Services](#services)
+  - [Pot transfer](#pot-transfer)
+  - [(Un-)Register Webhooks](#un-register-webhooks)
+
+{% include integrations/config_flow.md %}
+
+## Prerequisites and approval
+
+1. Before adding the Monzo integration, you'll need to create a [Monzo developer account](https://developers.monzo.com/). 
+2. From here, you need to create a new OAuth client for Home Assistant to use by going to **Clients** > **New OAuth Client**. 
+3. Then, fill in the form as follows:
+
+  - Name: Home Assistant
+  - Logo URL: This can be left blank
+  - Redirect URLs: <https://my.home-assistant.io/redirect/oauth>
+  - Description: For example: Used by the Monzo Home Assistant Integration
+  - Confidentiality: Confidential
+
```

Comment:
```suggestion
3. Then, fill in the form as follows:
   - Name: Home Assistant
   - Logo URL: This can be left blank
   - Redirect URLs: <https://my.home-assistant.io/redirect/oauth>
   - Description: For example: Used by the Monzo Home Assistant Integration
   - Confidentiality: Confidential

```